Description: don't bail plymouth-upstart-bridge if TERM is not set
 and/or running on a dummy terminal, as we can run on those.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

--- a/src/upstart-bridge/plymouth-upstart-bridge.c
+++ b/src/upstart-bridge/plymouth-upstart-bridge.c
@@ -307,7 +307,9 @@
   if (should_be_verbose && !ply_is_tracing ())
     ply_toggle_tracing ();
 
-  setupterm (NULL, STDOUT_FILENO, NULL);
+  /* Don't bail on dummy/hardcopy terminals */
+  int errret=0;
+  setupterm (NULL, STDOUT_FILENO, &errret);
 
   is_connected = ply_boot_client_connect (state.client,
                                           (ply_boot_client_disconnect_handler_t)
